shell/platform/darwin/ios/ios_context_metal.mm

```diff
@@ -34,6 +35,19 @@
     return;
   }
 
+  CVMetalTextureCacheRef texture_cache_raw = NULL;
+  auto cv_return = CVMetalTextureCacheCreate(kCFAllocatorDefault,  // allocator
+                                             NULL,           // cache attributes (NULL default)
```

Comment:
clang format doesn't line these up?

Reply:
Apparently not. I don't format code so its got to be `clang-format`. I think its because the comments are pushed up against the column limit.